server/types/models/video/video-playlist.ts

```diff
@@ -26,6 +30,10 @@ export type MVideoPlaylistWithElements =
   MVideoPlaylist &
   Use<'VideoPlaylistElements', MVideoPlaylistElementLight[]>
 
+export type MVideoPlaylistSummaryWithElements =
```

Comment:
Just noticed that `MVideoPlaylistWithElements` is not used anywhere
I think it's also the same for `MVideoPlaylistIdWithElements`